motor/core.py

```diff
@@ -547,6 +547,11 @@ def aggregate(self, pipeline, **kwargs):
             returning aggregate results using a cursor.
           - `collation` (optional): An instance of
             :class:`~pymongo.collation.Collation`.
+          - `let` (dict): A dict of parameter names and values. Values must be
+            constant or closed expressions that do not reference document
+            fields. Parameters can then be accessed as variables in an
+            aggregate expression context (e.g. ``"$$var"``). This option is
+            only supported on MongoDB >= 5.0.
```

Comment:
I copied this wording from the CRUD spec: https://github.com/mongodb/specifications/commit/3db387995bc0e64df862d621aaa31387afc1537c